.github/workflows/deploy.yml

```diff
@@ -0,0 +1,56 @@
+name: deploy
```

Comment:
Deploy is a little bit misleading. We only upload the docker image. Deploying would involve telling the server to pull the new docker image and restart the container. I would suggest doing so using Ansible. I can create a playbook for this soon.